tests/validation/autoscale_test.go

```diff
@@ -25,7 +25,7 @@ func autoscaleTests(t *testing.T, when spec.G, it spec.S) {
 		it("should autoscale down to 0", func() {
 			// Precondition
 			assert.True(t, service.IsReady())
-			assert.Equal(t, "Hello World", service.GetAppEndpointResponse())
+			assert.Equal(t, "Hi there, I am rio:production6", service.GetAppEndpointResponse())
```

Comment:
This is ok ?

Reply:
Yes... We ignore the no healthy upstream though as part of getting response:
```
	for i := 0; i < 5; i++ {
		if resp == "no healthy upstream" {
			time.Sleep(1 * time.Second)
			resp, _ = GetURL(endpoint)
		} else {
			break
		}
	}
```

Reply:
I meant `rio:production6` seems random

Reply:
oh we can blame @StrongMonkey for his autoscale image returning that